docs/pages/api.rst

```diff
@@ -249,27 +255,30 @@ Examples:
 ******
 MOSMIX
 ******
-::
+MOSMIX-S - less parameters:
+.. code-block:: python
 
-    from wetterdienst.mosmix.api import MOSMIXRequest
+    from wetterdienst.dwd.forecasts import DWDMosmixData, DWDFcstPeriodType
 
-    # MOSMIX-L, all parameters
-    mosmix = MOSMIXRequest(station_ids=["01001", "01008"])
-    response = mosmix.read_mosmix_l_latest()
+    mosmix = DWDMosmixData(
+        station_ids=["01001", "01008"],
+        period_type=DWDFcstPeriodType.FORECAST_LONG
```

Comment:
That might become `DWDMosmixPeriod.LONG`, which reads more fluently.

Reply:
Oh dang, I still have to change this! L does not refer to the forecast range but instead the number of parameters (-> LARGE)